### ISSUE TITLE
Enable timeout vibration when phone is locked

### ISSUE THREAD

**Issue by kazlauskis** [BiologicalRecordsCentre/ABLE#213]
It only vibrates when the user is in the app and the device is not locked.

**Comment by DavidRoy** [comment]
@kazlauskis is this requirement still open?

**Comment by kazlauskis** [comment]
Yes, this isn't done yet. When the app isn't active, it is then pushed to the background processes, which work very differently. At the moment, the app vibration works only in certain scenarios when the device is locked.

**Comment by kazlauskis** [comment]
https://github.com/BiologicalRecordsCentre/ABLE/issues/573

> Back from holidays, done a lot of 15 minute counts, and noticed that when the phone is in sleeping mode it DOES vibrate at the 3 minute warning, but NOT at the end of the count. I use a Asus Zenfone 8 with Android 13. Would be great if the vibration could come to the end of the count as well. @kazlauskis
